### PR TITLE
{lang,tools}[dummy] travis v1.8.2, Ruby v2.3.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~https://github.com/hpcugent/easybuild-easyblocks/pull/901~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2107.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4d82ebfa3811729b3d1f828493b84eb3 for a full test report.

**Comment by boegel** [comment]
@rjeschmi please review/test? note that this requires https://github.com/hpcugent/easybuild-easyblocks/pull/901

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/39f2b254d48e12c8379d90bfa191a439 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2578.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2d31a7f6973066954dacd0513831b4f6 for a full test report.

**Comment by boegel** [comment]
@wpoely86, @rjeschmi: please review?

Th bug fix from https://github.com/hpcugent/easybuild-framework/pull/1748 is required to install both easyconfigs in here correctly using current framework `develop`...

File: easybuild/easyconfigs/r/Ruby/Ruby-2.3.1.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#    With <3 for EasyBuild
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/Ruby/Ruby-2.3.1.eb]: we should add this in the template :smile:

File: easybuild/easyconfigs/r/Ruby/Ruby-2.3.1.eb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+name = 'Ruby'
+version = '2.3.1'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/Ruby/Ruby-2.3.1.eb]: no compile?

Comment (boegel) [inline easybuild/easyconfigs/r/Ruby/Ruby-2.3.1.eb]: Yes, but my goal was to obtain a `travis` module that I could use whenever, so regardless of which toolchain is loaded...

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**